resources/js/comment.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function(){
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        }
+    });
+
+    $('.site-btn').click(function(e) {
+        e.preventDefault();
+        var _this = $(this);
+        var product_id = $(this).data('product-id');
+        var url = '/comments/' + product_id;
+
+        var data = {
+            'product_id': product_id,
+            'content' : $('textarea[name="content"]').val(),
+        };
+
+        $.ajax({
+            url: url,
+            type: 'POST',
+            data: data,
+
+            success: function(result) {
+                var html =
+                `<div class="comment-option">
+                    <div class="co-item">
+                        <div class="avatar-pic">
+                            <img src="` + result.image + `" alt="">
```

Comment:
```suggestion
                            <img src="${result.image}" alt="">
```
như này được ko em?
https://viblo.asia/p/es6-es7-la-gi-va-nhung-tinh-nang-cu-phap-hay-dung-trong-es6-Eb85owmk52G#_24-template-literals-5